src/test/java/io/horrorshow/codey/time/DiscordTimezoneTest.java

```diff
@@ -0,0 +1,78 @@
+package io.horrorshow.codey.time;
+
+import io.horrorshow.codey.discordutil.CodeyConfig;
+import io.horrorshow.codey.discordutil.DiscordMessage;
+import io.horrorshow.codey.discordutil.DiscordUtils;
+import io.horrorshow.codey.formatter.DiscordCodeFormatter;
+import io.horrorshow.codey.formatter.JavaFormatter;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Message;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.time.DayOfWeek;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+class DiscordTimezoneTest {
+    @Mock
+    JDA jda;
+    CodeyConfig codeyConfig;
+    DiscordUtils utils;
+    DiscordTimezone discordTimezone;
+
+    @BeforeEach
+    void init() {
+        MockitoAnnotations.openMocks(this);
+        codeyConfig = new CodeyConfig();
+        utils = new DiscordUtils(jda, codeyConfig);
+        discordTimezone = new DiscordTimezone(jda, utils);
+    }
+
+    @Test
+    void test_no_weekday()
+    {
+        String[] tests = {
```

Comment:
I don't see a reason why there needs to be a array of strings, it's more readable if you have them directly in the assertions, for example
```java
assertThat(foo("18:00 Europe/Amsterdam")).isEqualTo(...)
```